src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -156,7 +199,13 @@ public class PullRequestHandler : IHttpHandler
         return success;
     }
 
-    private bool SyncToPr(uint pr, StringBuilder executionResult)
+    private string GetRepoPath(uint repoId)
```

Comment:
Eventually, we have to rely on ACL to enforce the security, which is what we already did, but it's not a bad idea to have some sanity validation like rejecting anything contains .., / and \\.

I do have a user scenario related to private Helix run. I want `hr.ashx?id=Tom&pr=123` to work when Tom try to submit a private Helix run. This change is blocking my scenario 😄.